integration-tests/tests/pages/review.page.ts

```diff
@@ -1,8 +1,18 @@
-import { Page } from '@playwright/test';
+import { Page, expect } from '@playwright/test';
 
 export class ReviewPage {
     private page: Page;
 
+
+    private viewSequencesButton = () =>
+        this.page.getByTestId(/view-sequences-/).first();
+    private sequencesDialog = () =>
+        this.page.locator('div:has(> div > h2:text("Processed Sequences"))').first();
+    private sequencesDialogCloseButton = () =>
+        this.sequencesDialog().getByRole('button', { name: '✕' });
+    private sequenceViewerContent = () => this.page.getByTestId('fixed-length-text-viewer');
+    private sequenceTabs = () => this.page.locator('.tab');
```

Comment:
This selector is very broad. would `this.sequencesDialog().locator('.tab')` also work? Not blocking.

Reply:
Resolving this for now as I don't know the answer and would prefer to avoid another test cycle